iron-collapse.html

```diff
@@ -111,6 +111,7 @@
 
     hostAttributes: {
       role: 'group',
+      'aria-hidden': 'true',
```

Comment:
Without it, `aria-hidden` was set only when `_openChanged` was called.
Like this, we'll have `aria-hidden` correctly set when the element is created.
Also for coherency, we were setting the `aria-expanded` on the line below, so why not setting also the `aria-hidden` here? ;)